# Define the goal
float32 x
float32 y
float32 yaw
float32 x_err_ss
float32 y_err_ss
float32 yaw_err_ss
float32 timeout_s
---
# Define the result
float32 x
float32 y
float32 yaw
float32 duration_s
# mission status
uint8 status
uint8 SUCCEEDED  = 0
uint8 PREEMPTED  = 1
uint8 TIMEOUT    = 2